playwright-e2e/README.md

```diff
@@ -49,20 +49,23 @@ Note: Update docker image version when upgrading Playwright version
 - Change dir to **/playwright-e2e**
   > cd playwright-e2e/
 
-- Start Playwright docker image
-  > docker run -v $PWD:/e2e -w /e2e -it --rm --ipc=host mcr.microsoft.com/playwright:v1.25.0-focal /bin/bash
+- Start running Playwright docker image
+  > docker run -v $PWD:/e2e -w /e2e -it --rm -d --ipc=host -p 9323:9323 mcr.microsoft.com/playwright:v1.28.0-focal /bin/bash
```

Comment:
Allow html test result inside docker container to be opened outside docker container.